### PR TITLE
fix a typo error in description

### PR DESCRIPTION
**Pull request by WanLinghao**
**What this PR does / why we need it**:
fix a typo error in description
**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: pkg/kubectl/cmd/create/create_priorityclass.go [hunk 1]
```diff
@@ -32,7 +32,7 @@ var (
 
 	pcExample = templates.Examples(i18n.T(`
 		# Create a priorityclass named high-priority
-		kubectl create priorityclass default-priority --value=1000 --description="high priority"
+		kubectl create priorityclass high-priority --value=1000 --description="high priority"
```
Comment (nikhita) [inline pkg/kubectl/cmd/create/create_priorityclass.go]: :+1: thanks!

**Comment by nikhita** [comment]
/assign @soltysh 
for approval

**Review by soltysh — APPROVED** [review]
/approve

**Pull request merged**